neurom/features/neuritefunc.py

```diff
@@ -244,6 +244,21 @@ def _seg_taper_rates(sec):
     return map_segments(_seg_taper_rates, neurites, neurite_type)
 
 
+def section_taper_rate(neurites, neurite_type=NeuriteType.all):
+    '''taper rate of the sections in a collection of neurites
+
+    Taper rate is defined here as the linear fit along a section'''
```

Comment:
Please specify direction (from root to tips) and expected sign (expected to be <=0 for neurons) to clarify contradicting definitions for this one.